src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -680,18 +680,24 @@ protected virtual LoadoutItemWithTargetPath.ReadOnly SelectWinningFile(IEnumerab
     {
         return files.MaxBy(GetPriority);
 
-        int GetPriority(LoadoutItemWithTargetPath.ReadOnly item)
+        // Placeholder for a more advanced selection algorithm
+        long GetPriority(LoadoutItemWithTargetPath.ReadOnly item)
         {
             foreach (var parent in item.AsLoadoutItem().GetThisAndParents())
             {
                 if (!parent.TryGetAsLoadoutItemGroup(out var group))
                     continue;
-
+                
+                // GameFiles always lose
                 if (group.TryGetAsLoadoutGameFilesGroup(out var gameFilesGroup))
                     return 0;
+                
+                // Return a placeholder priority based on creation time of the LoadoutGroup, newest wins.
+                // This allows for some degree of control and predictability in the selection process.
+                return group.GetCreatedAt().ToUnixTimeSeconds();
             }
-
-            return 50;
+    
+            return int.MaxValue;
```

Comment:
Why are you returning `int.MaxValue`?

Reply:
It was supposed to handle Override. I changed the code to handle things better